src/lib/containers/download_list_v2/MeetAccessRequirementCard.tsx

```diff
@@ -96,7 +96,7 @@ export const MeetAccessRequirementCard:React.FunctionComponent<MeetAccessRequire
     <>
       {!isError && !isFetching && content}
       {isFetching && (
-        <LoadingMeetAccessRequirementCard />
+        <LoadingAccessRequirementCard />
```

Comment:
Rename LoadingMeetAccessRequirementCard to LoadingAccessRequirementCard, and re-use in RequestDownloadCard as well.